tests/ieee802154_submac/Makefile

```diff
@@ -20,8 +20,12 @@ USEMODULE += od
 USEMODULE += shell
 USEMODULE += ps
 USEMODULE += event_thread_highest
-USEMODULE += netdev_ieee802154_submac
 USEMODULE += netdev_default
+USEMODULE += luid
+USEMODULE += eui_provider
+USEMODULE += ieee802154
+USEMODULE += ieee802154_submac
```

Comment:
Shouldn't there be a dependency between these two somewhere?

Reply:
actually there's no dependency. As it is, `ieee802154` only defines header helpers, which are not used by the SubMAC.
IMO we should rename `ieee802154` to `ieee802154_hdr` or something like that